README.rst

```diff
@@ -70,3 +70,41 @@ To generate correct files, please provide the following input to Cookiecutter:
                                  super project of this module, e.g. Invenio
                                  itself.
 ================================ =============================================
+
+Usage example
+--------------
+This example might be deprecated at the time you read it, but it will give you
+a good idea on how to use this template to generate a new project.
+
+Requirements:
+==============
+You will need to have installed the Cookiecutter tool, we recommend to use a
+virtualenv, specially with virtualenvwrapper to help you around. So assuming
+that you have virtualenvwrapper already installed (remember to add `source
+virtualenvwrapper.sh` to your bashrc and relod the console or source it
+directly after installing it), you can just run::
+
+    $ mkvirtualenv cookiecutter
```

Comment:
Is it necessary? the `::` option is essentially the same (without syntax highlight, though I don't think it helps here) and way easier to read and write on the text file.